test/spec.js

```diff
@@ -105,4 +105,48 @@ describe('spec', () => {
       done();
     });
   });
+
+  it('should support transforming the request/response', (done) => {
+    let request, response;
+    const event = {}
+    const context = {}
+
+    const handler = serverless((req, res) => {
+      request = req;
+      response = res;
+      res.end('');
+    }, {
+      transform: (evt, ctx) => (req, res) => {
```

Comment:
I just thought it made for a clearer separation between AWS parameters and the HTTP objects. I figured since most use cases would probably be using only event + req or res, it also allows to skip the other 2 (i.e. `event => req => { }`). Otherwise it's functionally the same.